composer.json

```diff
@@ -13,6 +13,23 @@
     {
       "type": "composer",
       "url": "https://packages.drupal.org/8"
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "drupal/lb_ux",
+        "version": "dev-8.x-1.x",
+        "type": "drupal-module",
+        "source": {
+          "url": "https://git.drupalcode.org/project/lb_ux.git",
+          "type": "git",
+          "reference": "8.x-1.x"
+        }
+      }
+    },
+    {
+      "type": "vcs",
+      "url": "https://git.drupalcode.org/project/panels_everywhere.git"
```

Comment:
why do you describe these 2 modules in different way?

Reply:
`vcs` works for repos containing a `composer.json` file but `lb_ux` didn't contain one, however I agree that it might be better to have both in a similar way